.pylintrc

```diff
@@ -0,0 +1,645 @@
+[MAIN]
+
+# Analyse import fallback blocks. This can be used to support both Python 2 and
+# 3 compatible code, which means that the block might have code that exists
+# only in one or another interpreter, leading to false positives when analysed.
+analyse-fallback-blocks=no
+
+# Clear in-memory caches upon conclusion of linting. Useful if running pylint
+# in a server-like mode.
+clear-cache-post-run=no
+
+# Load and enable all available extensions. Use --list-extensions to see a list
+# all available extensions.
+#enable-all-extensions=
```

Comment:
There are a handful of times in this file where there are parameters commented out or left with no value associated -- are these necessary? I would be inclined to remove anything in here that is not explicitly required to avoid bloat.

Reply:
This file was automatically-generated, and so includes a lot of commented-out defaults. I don't feel we need to keep those pieces, personally.